docs/modules/certification.md

```diff
@@ -0,0 +1,139 @@
+metadata](https://github.com/NethServer/ns8-nethforge/) starts the
+Community certification process for the application.
+
+After approval, at any time, the application can be removed from NethForge
+by the NethForge repository maintainers. For example, this can happen if
+the current version is no longer maintained but has significant security
+issues, or if its behavior is deemed malicious.
+
+Here follow some tips that may help to quickly complete the certification
+process.
+
+### 1. Metadata and license
+
+0. The PR for NethForge must contain a directory and files as described in
+   [Create a
+   repository](../../core/software_repositories#create-a-repository).
+
+0. Similarly, fill the [metadata.json](../metadata/) file of the app UI
+   with the required information. Provide details for reporting bugs, such
+   as a link to the app's bug tracker (you can link to
+   `https://github.com/NethServer/dev/issues`), the developer's email, and
+   any other relevant contact information.
+
+0. Additionally, in the `metadata.json`, consider linking a document via
+   `terms_url` that clarifies any data treatment or application behavior
+   the user should be aware of. This is particularly important if the
+   application handles sensitive data or must conform to data privacy
+   laws. During the certification process, we will pay close attention to
+   compliance with EU laws and regulations.
+
+0. Administrator documentation. This is the documentation for the
+   administrator who must configure the applications for the final users.
+   A public web page link must be provided in the PR description. You can
+   decide to write the documentation on your preferred web site or to
+   directly create a new [NS8 Admin's
+   Manual](https://github.com/NethServer/ns8-docs/) page with a PR. In
+   both cases the metadata.json documentation URL will point to the latter
+   with an application summary available from there.
+
+0. Integrate the `README.md` file in the code repository with
+   documentation for developers and the support team. A detailed
+   description of container volumes, environment variables, and special
+   maintenance or customization procedures is very important. If something
+   is still missing, document it for instance under a "Roadmap",
+   "Limitations", "TODO" section. Provide also a link to your Community
+   profile to contact you.
+
+0. The application and its parts must have no licensing conflicts.
+   Licenses must permit unlimited distribution of the application.
+
+### 2. Platform compliancy
+
+0. The implementation of platform-defined behaviors, where present, must work
+   correctly. Pay attention to:
+
+    - Application Creation and Removal
+    - Clone and Move
+    - Backup and Restore
+    - Change of cluster Email notification settings
+    - HTTP routes management
+    - TLS certificates management
+    - Bind to one or more User Domains
+    - Node firewall management
+    - Node ports allocation
```

Comment:
I will remove these steps because not every module will need to use or implement these features. Instead, I suggest making the correct implementation of the following features mandatory:
- Installation/Removal
- Updates
- Backup/restore
- Clone/move